homepage/homepage.go

```diff
@@ -50,7 +50,7 @@ func handle(w http.ResponseWriter, req *http.Request, cfg *config.Config, userAc
 	}
 
 	basePage := rend.NewBasePageModel()
-	m := mapper.Homepage(lang, basePage, homepageContent.MainFigures, homepageContent.FeaturedContent, homepageContent.AroundONS, homepageContent.ServiceMessage, homepageContent.EmergencyBanner)
+	m := mapper.Homepage(config.Config{}, lang, basePage, homepageContent.MainFigures, homepageContent.FeaturedContent, homepageContent.AroundONS, homepageContent.ServiceMessage, homepageContent.EmergencyBanner)
```

Comment:
i think they should be the config we get in main.go https://github.com/ONSdigital/dp-frontend-homepage-controller/blob/develop/main.go#L45

it'll need to be passed through to this func